namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-production/resources/publisher.tf

```diff
@@ -0,0 +1,56 @@
+##################################################
+# Publisher RDS
+
+module "publisher-rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=3.0"
+
+  cluster_name               = "${var.cluster_name}"
+  cluster_state_bucket       = "${var.cluster_state_bucket}"
+  db_backup_retention_period = "2"
+  application                = "formbuilderpublisher"
+  environment-name           = "${var.environment-name}"
+  is-production              = "${var.is-production}"
+  infrastructure-support     = "${var.infrastructure-support}"
+  team_name                  = "${var.team_name}"
+}
+
+resource "kubernetes_secret" "publisher-rds-instance" {
+  metadata {
+    name      = "rds-instance-formbuilder-publisher-production"
+    namespace = "formbuilder-platform-production"
+  }
+
+  data {
+    # postgres://USER:PASSWORD@HOST:PORT/NAME
+    url = "postgres://${module.publisher-rds-instance.database_username}:${module.publisher-rds-instance.database_password}@${module.publisher-rds-instance.rds_instance_endpoint}/${module.publisher-rds-instance.database_name}"
+  }
+}
+
+##################################################
+
+########################################################
+# Publisher Elasticache Redis (for resque + job logging)
+module "publisher-elasticache" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=2.0"
```

Comment:
Can you please update to version `2.1` for the elasticache cluster module?

Reply:
Done